python/lsst/ip/diffim/imagePsfMatch.py

```diff
@@ -322,14 +322,6 @@ def __init__(self, *args, **kwargs):
         self.makeSubtask("selectDetection", schema=self.selectSchema)
         self.makeSubtask("selectMeasurement", schema=self.selectSchema, algMetadata=self.selectAlgMetadata)
 
-    def getFwhmPix(self, psf, position=None):
```

Comment:
It looks like one unit test fails if you remove this: `tests/test_imagePsfMatch.py::PsfMatchTestCases::testSubtractExposures`
(or could be fixed by replacing instances of `self.getFwhmPix` in `imagePsfMatch.py` with `pDiffim.utils.getPsfFwhm`)

Reply:
This didn't show up on Github search. I've fixed this now.